### PR TITLE
Fix warnings shown during tests

### PR DESCRIPTION
**Pull request by aperezhortal**
This PR fixes some of the warnings shown when the test suite is run. 
- `pysteps.motion.vet`: Fix bug in `vet` keywords handling. Previously, the `method` argument passed to the minimizer was ignored.
- `pysteps.io.importers`: Add a function to safely get the rain/no rain threshold that handles NaN correctly.
- Fix collections imports. Remove DeprecationWarning for using the ABCs from 'collections' instead
    of from 'collections.abc'.
- `pysteps.timeseries.autoregression`: Make some docstrings raw literals, avoiding the interpretation of math equations as invalid escape sequences.

### CODE REVIEW

File: pysteps/io/importers.py [hunk 1]
```diff
@@ -200,6 +200,26 @@ def _get_grib_projection(grib_msg):
...
+
+    Returns
+    -------
+
+    threshold : float
+    """
+    valid_mask = np.isfinite(precip)
+    if valid_mask.any():
+        _precip = precip[valid_mask]
+        return np.min(_precip[_precip > _precip.min()])
```
Comment (dnerini) [inline pysteps/io/importers.py]: Line 218 will fail if precip is a uniform field (e.g. all zeros).
I think we need to add an additional test to line 216, something like:
`if valid_mask.any() and (precip[valid_mask] > precip[valid_mask].min()).any():`
or
`if valid_mask.any() and not (precip[valid_mask] == precip[valid_mask][0]).all():`
or whatever is more elegant/efficient, you get the point :-)

**Review by dnerini — CHANGES_REQUESTED** [review]
Turns out there was a bit of a mess on how we defined the rainfall threshold in the various importers, thanks for cleaning this up! 
As explained in my in-line comment, I think we just need to handle cases when the field has uniform value (e.g. all zeros), otherwise looks good to go!

File: pysteps/io/importers.py (hunk 1 above)
Comment (aperezhortal) [reply pysteps/io/importers.py]: Commit tf4614c0 fixes that bug :bowtie:

**Review by dnerini — APPROVED** [review]
excellent, looks good to go!

**Pull request merged**